### PR TITLE
Checkpoint 12

### PR DESCRIPTION
**Pull request by impactmass**
# Preview: 👉 [here](https://vkyi2.csb.app/)

### CODE REVIEW

File: checkpoint/contact.html [hunk 1]
```diff
@@ -26,8 +40,7 @@
 
     <br />
     <input type="submit" value="Submit" />
-
   </form>
 </body>
 
-</html>
+</html>s
```
Comment (impactmass) [inline checkpoint/contact.html:46]:
```suggestion
</html>
```

File: checkpoint/css/main.css [hunk 2]
```diff
@@ -68,36 +73,92 @@ a:visited {
...
- */
+* {
+  box-sizing: border-box;
+}
+
+/* nav styles */
+nav {
+  margin-bottom: 50px;
+}
+
```
Comment (impactmass) [inline checkpoint/css/main.css:117]:
```suggestion
```

File: checkpoint/css/main.css [hunk 3]
```diff
@@ -68,36 +73,88 @@ a:visited {
...
+.logo {
+  float: left;
+}
+
+.main-menu {
+  float: right;
+}
+
+.main-menu li {
+  float: left;
```
Comment (impactmass) [inline checkpoint/css/main.css:137]:
```suggestion
  display: inline-block;
```

File: checkpoint/index.html [hunk 4]
```diff
@@ -4,26 +4,46 @@
 <head>
   <meta charset="UTF-8" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>ReDI Student Portfolio</title>
-  <link href="css/main.css" rel="stylesheet">
+  <title>Joanna Ahmed</title>
```
Comment (impactmass) [inline checkpoint/index.html:7]:
```suggestion
  <title>Joanna Ahmed | Web Developer</title>
```

File: checkpoint/css/main.css [hunk 5]
```diff
@@ -68,36 +73,88 @@ a:visited {
...
+h3 {
+  font-family: "Julius Sans One", sans-serif;
+}
+
+h1 {
+  font-size: 3.2em;
+}
+
 h2 {
   font-size: 35px;
```
Comment (impactmass) [inline checkpoint/css/main.css:93]:
```suggestion
```

File: checkpoint/css/main.css [hunk 6]
```diff
@@ -68,36 +73,88 @@ a:visited {
...
+  font-size: 1.6em;
 }
 
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (impactmass) [inline checkpoint/css/main.css:99]:
```suggestion
```

File: checkpoint/css/main.css [hunk 7]
```diff
@@ -88,16 +104,58 @@ h2 {
...
+
+.clear-fix {
+  clear: both;
+}
+/* end nav styles */
+
+/* header styles */
+header {
+  color: rgb(20, 20, 20);
+}
```
Comment (impactmass) [inline checkpoint/css/main.css:149]:
```suggestion
```

File: checkpoint/css/main.css [hunk 8]
```diff
@@ -88,16 +104,55 @@ h2 {
...
+  display: inline-block;
+}
+
+.clear-fix {
+  clear: both;
+}
+/* end nav styles */
+
+/* header styles */
+
```
Comment (impactmass) [inline checkpoint/css/main.css:147]:
```suggestion
```

**Comment by impactmass** [comment]
@angelsvirkov please take a look.

File: checkpoint/css/main.css [hunk 9]
```diff
@@ -88,16 +104,54 @@ h2 {
...
-  border: 5px groove paleturquoise;
   border-radius: 20px;
 }
 
-/*
- * Lesson: CSS Units
- * Center the profile image container and make its size 20% relative to width of it's parent
- */
+* {
+  box-sizing: border-box;
```
Comment (angelsvirkov) [inline checkpoint/css/main.css:111]: As agreed, make sure students have read this at home, as I gave it as a homework

**Review by angelsvirkov — APPROVED** [review]
The website looks awesome. The changes are good. One concern I have is whether students will be able to finish all of those in class or at home. What is your plan here? Would you try to introduce all changes here or would you leave all of this as a homework (maybe together with the diff, so students know what needs to change)? Besides. PR is approved

File: checkpoint/css/main.css (hunk 9 above)
Comment (impactmass) [reply checkpoint/css/main.css:111]: Yes, sounds good. I will check that they read about it. And also explain it again if needed.

**Comment by impactmass** [comment]
> Would you try to introduce all changes here or would you leave all of this as a homework (maybe together with the diff, so students know what needs to change)?

Good point. I do not think we can cover the whole change shown here in the class. Especially since I'm doing a review before going into the day's topic. I will show them this checkpoint branch at the end, so they can finish up the remaining on their own time.

**Comment by impactmass** [comment]
closing as per the agreed checkpoint review process.

**Pull request closed without merging**